src/test/java/org/broadinstitute/hellbender/engine/GATKToolUnitTest.java

```diff
@@ -58,6 +62,31 @@ public void traverse() {
         }
     }
 
+    @CommandLineProgramProperties(
+            summary = "TestGATKToolValidationStringency",
+            oneLineSummary = "TestGATKToolValidationStringency",
+            programGroup = ReadProgramGroup.class
```

Comment:
No, the test tools here don't appear to show up, though I'm not sure why. @lbergelson do you know why this is ? I could make a separate test group that implements omitFromCommandLine, but it doesn't appear to be necessary in this case.